docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -184,10 +186,65 @@ To apply or remove alert tags on individual alerts, do one of the following:
 
 To apply or remove alert tags on multiple alerts, select the alerts you want to change, then click *Selected _x_ alerts* at the upper-left above the table. Click *Apply alert tags*, select or unselect tags, then click *Apply tags*. 
 
-
 [role="screenshot"]
 image::images/bulk-apply-alert-tag.png[Bulk action menu with multiple alerts selected, 450]
 
+[float]
+[[assign-users-to-alerts]]
+==== Assign users to alerts
+
+Assign users to alerts that you want them to investigate, and manage alert assignees throughout an alert's lifecycle.  
+
+IMPORTANT: Users are not notified when they've been assigned to, or unassigned from, alerts.  
+
+|==============================================
+| Action | Instructions 
+
+| Assign users to an alert 
+
+a| Choose one of the following:
+
+* **Alerts table** - Click **More actions** (**...**) in an alert's row, then click **Assign alert**. Select users, then click **Apply**. 
+* **Alert details flyout** - Click **Take action -> Assign alert**. Alternatively, click the **Assign alert** icon at the top of the alert details flyout, select users, then click **Apply**. 
+
+NOTE: Users assigned to some of the selected alerts will be displayed as unassigned in the selection list. Selecting said users will assign them to all alerts they haven't been assigned to yet.
```

Comment:
Revisiting this and the demo [here](https://github.com/elastic/staging-serverless-security-docs/pull/232#issuecomment-1834275306), this note does actually belong two rows down. I'll move it there - thanks, @natasha-moore-elastic! 

@e40pud looks like this is still happening in 8.12 BC3. Should we doc this bug as a known issue in 8.12 or do you think it'll be fixed before 8.12 GAs? No worries if there's no plans to address this in 8.12. I saw that you had filed https://github.com/elastic/security-team/issues/8008 and https://github.com/elastic/kibana/issues/172821 to track work on the bug so I figured I'd ask in case something was in the works.

Reply:
Note moved in [7dffed1](https://github.com/elastic/security-docs/pull/4476/commits/7dffed18d107d753df52bd0aaa4fb3f28519c7de).

Reply:
This issue won't be addressed in `8.12`. We are dependent on another team and we will need to sync with them to understand when they will be able to prioritise the fix.

Reply:
Thanks for clarifying! Did you want to call this out in the release notes as a known issue in 8.12 or is this note in the docs sufficient for communicating this information to users? I'm would recommend the latter since it's a UI limitation that doesn't have a clear fix on the horizon, but I'll defer to you. :)

Reply:
Agree, this note will be sufficient enough in my opinion.